studio/tsconfig.test.json

```diff
@@ -1,6 +1,8 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    // minimal target of es6 required for extending Event using es6 class features, see radix setup file (tests/setup/radix.js)
```

Comment:
This looks like invalid JSON - should it be a JS file?

Reply:
ah nice catch, ts-jest seems to parse it fine, but i'll remove it anyway.